modules/ROOT/pages/cli-commands-troubleshoot.adoc

```diff
@@ -43,7 +43,7 @@ If you are uploading multiple files, omit the `--type` flag to let Anypoint CLI
 Verify that the `classifier.packaging` and file name values in your `--files` flag options are correct and that they match the asset type you specified in the `--type` flag. 
 * Include properties in `exchange.json` or `--properties` that are incorrect.
 +
-The `exchange.json` file is optional when uploading an asset because the required properties are either included or are inferred from parameters, flags, or options. If you download a ruleset project on which to base your custom ruleset, do one of the following before uploading your custom ruleset:
+The `exchange.json` file is optional when uploading an asset because the CLI includes or infers required properties from parameters, flags, and options. If you download a ruleset project on which to base your custom ruleset, do one of the following before uploading your custom ruleset:
```

Comment:
```suggestion
The `exchange.json` file is optional when uploading an asset because the CLI includes or infers required properties from parameters, flags, and options. If you download a ruleset project to base your custom ruleset on, do one of the following before uploading your custom ruleset:
```